CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The changelog is available [on GitHub][2].
 
 ## Unreleased: 0.6.0.0
 
+* [#194](https://github.com/kowainik/relude/pull/194):
+  Use `mapToFst` instead of `zip` to improve list fusion.
```

Comment:
Can we specify the function name, so it's easier to parse where users can expect the changes :slightly_smiling_face:

Reply:
Done.